core/src/main/java/com/datastax/oss/driver/internal/core/metrics/NoopNodeMetricUpdater.java

```diff
@@ -22,6 +22,10 @@
 @ThreadSafe
 public class NoopNodeMetricUpdater implements NodeMetricUpdater {
 
+  public static NoopNodeMetricUpdater INSTANCE = new NoopNodeMetricUpdater();
```

Comment:
perhaps this is not necessary, however previously we were creating 1 of these per node (if metric registry was null).  With the change to create this in `ChannelFactory` we'd create one per connection so thought it might be good to make this a singleton.

Reply:
👍 wfm